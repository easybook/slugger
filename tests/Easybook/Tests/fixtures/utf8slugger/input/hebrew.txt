׆אבגדהוזחטיךכלםמן
נסעףפץצקרשת
װױײ
ﬠﬡﬢﬣﬤﬥﬦﬧﬨ﬩שׁשׂשּׁשּׂאַאָ
אּבּגּדּהּוּזּטיּּךּכּלּמּ
נּסּףּפּצּקּרּשּתּוֹבֿכֿפֿאל