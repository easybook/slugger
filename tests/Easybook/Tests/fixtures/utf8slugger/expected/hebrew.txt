bgdhwzhtykklmmn
nsppzzqrst
wwwyyy
dhklmrt-ssss
bgdhwztykklm
nsppzqrstwbkpl